products/opensuse/main.pm

```diff
@@ -750,6 +750,7 @@ elsif (get_var("WICKED")) {
 }
 elsif (get_var("REGRESSION")) {
     if (check_var("REGRESSION", "installation")) {
+        set_var('NOAUTOLOGIN', 1);
```

Comment:
I don't know if this should be here. NOAUTOLOGIN so far has been a setting based on the medium, e.g. for SLES it has been set, IIRC for SLED not, and for openSUSE … I don't know. But with setting his hardcoded here it won't be possible to change. Please crosscheck existing uses of this variable. Maybe it should just be set at the top of products/opensuse/main.pm in all cases

Reply:
- NOAUTOLOGIN is set as a SLE specific variables in /sle/main.pm (for all cases: SLES, SLED...)
- Currently it is only set for minimalx DESKTOP in /opensuse/main.pm and also set in four testsuites on o.o.o

We should set NOAUTOLOGIN to disable auto-login during the installation since there are several regression-* cases have the scenarios of switching user, changing password, switching desktop session...

But I am not sure this variable should be set at the top of /opensuse/main.pm, if you think it's not proper to set it in this block we can move it to the testsuite of regression-installation